web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -449,15 +449,15 @@ export default ({ settings }: any) => {
   )
   const [result, setResult] = useState<{ [key: string]: any }>({})
   const [queryError, setQueryError] = useState('')
-  const [dataSources, setDataSources] = useState<DataSources>([])
+  const [dataSources, setDataSources] = useState<TDataSources>([])
   const { token } = useContext(AuthContext)
   const dmssAPI = new DmssAPI(token)
 
   useEffect(() => {
     dmssAPI
       .dataSourceGetAll()
       .then((response: any) => {
```

Comment:
```suggestion
      .then((response: AxiosResponse<>) => {
```
Not directly related to this PR, but perhaps we can specify the specific type here as well? `AxiosResponse<CantRecallWhatsSupposedToBeHere>`?